### DISCUSSION TITLE
Quick switch between users

### DISCUSSION THREAD

**Discussion by kajvans** [jellyfin/jellyfin#11194] [Q&A]
How do I make it so that I can easily switch between users so that on my tv you don't have problems with series progression and more like that. Something like Netflix would be nice but if I could stream from my phone without problem then it is also fine.

**Comment by alvitali** [comment]
I think the easiest supported way in Jellyfin would be to just unhide all local users using the tv from the login screen and then use something like Quickconnect to login with a PIN code instead of having to type in the entire password.

Personally I've set it up so that every user has their own account for our Apple TV, and then we just switch accounts on the OS itself, instead of Jellyfin. This is definitely possible for Android TV and others as well. If the tv doesn't support that, it might warrant to buy sth inexpensive like a Chromecast, which is well worth it imo.

Before that, I just had one "tv" user account signed into the app on our tv, which everyone used. You'd miss your watching history when using the tv, but at least not get your history messed up by others watching with your account.

Anybody correct me if there are better options.

**Comment by kiwichick888** [comment]
I think it's so silly that Android mobile is such a pain to switch users. Sign out, select server, connect, enter username and password, sign in 🙄 Five steps to do something that should take two steps: switch user, select user.

**Reply by Abateman121** [reply]
This feature is the biggest hurdle for me switching to Jellyfin
